protocol/0053-PERP-product_builtin_perpetual_future.md

```diff
@@ -100,12 +100,7 @@ Every time a [mark to market settlement](./0003-MTMK-mark_to_market_settlement.m
 
 When the `settlement_schedule` event is received we need to calculate the funding payment. Store the current vega time as `funding_period_end`.
 
-Skip the funding payment calculation (set payment to `0`) if any of the following conditions is met:
-
-- no spot (external) data has been ingested since market was created,
-- market has been in auction throughout the entire funding period.
-
-Please refer to the following subsections for the details of calculation of the funding payment if none of the above conditions are met.
+Skip the funding payment calculation (set payment to `0`) if no spot (external) data has been ingested since market was create, otherwise calculate the funding payment as outlined below.
```

Comment:
```suggestion
Skip the funding payment calculation (set payment to `0`) if no spot (external) data has been ingested since market was created, otherwise calculate the funding payment as outlined below.
```